messaging/methods/media/listMedia.md

```diff
@@ -100,4 +120,24 @@ Continuation-Token: 678910
   }
 ]
 ```
+
+{% sample lang="csharp" %}
+
+```csharp
+var response = msgClient.ListMedia(MSG_ACCOUNT_ID, continuationToken);
+List<Media> mediaList = response.Data;
+```
+
+
+{% sample lang="ruby" %}
+
+```ruby
+media = messaging_client.list_media(MESSAGING_ACCOUNT_ID, continuation_token: "token")
+```
+
+{% sample lang="python" %}
+
+```python
+media = messaging_client.list_media(MESSAGING_ACCOUNT_ID, continuation_token="token")
+```
```

Comment:
Should we update these to look more like the C# one? I kinda short-handed them because the previous media example already showed how to use the data but I like consistency and can change it if desired

Reply:
I think short hand for ruby and python is ok.  Me and Dan decided to leave C# in a more expanded way because C#'s shorthand is newer and we wanted to not be teaching C# but bandwidth stuff only.

Reply:
Sounds good, works for me